cli/source/constants.ts

```diff
@@ -32,6 +32,10 @@ export enum LogSource {
   DockerWatchProcessIdLock
 }
 
+export enum ErrorNames {
+  NotFound = '-2'
+}
```

Comment:
We can have better naming here, maybe something like:
```suggestion
export enum ErrorCodes {
  SpawnProcessCommandNotFound = '-2'
}
```

Reply:
Good suggestion, @adnanhashmi09

Reply:
Sure, this looks better